ui/src/reportAnIssue.ts

```diff
@@ -33,22 +41,30 @@ Product: ${vscode.env.appName}
 Product Version: ${vscode.version}
 Language: ${vscode.env.language}`;
 
-    if (parsedError.stack) {
-        body = body.concat(`
+    // Try repeatedly with fewer lines of stack until we have a URL of a reasonable length
+    // tslint:disable-next-line: no-increment-decrement
```

Comment:
Rather than disable the rule, the [suggested fix](https://palantir.github.io/tslint/rules/increment-decrement/) is super simple: `stackLines -= 1`

Reply:
Stupid rules.